earth_enterprise/src/fusion/fusionui/AssetChooser.h

```diff
@@ -72,7 +72,7 @@ class AssetChooser : public AssetChooserBase {
 
   // from QWidget
   virtual void keyPressEvent(QKeyEvent* e);
```

Comment:
Since they are victuals, they probably should be made "final" to assure they match a base class virtual.